packages/airnode-utilities/src/gas-prices/gas-prices.ts

```diff
@@ -1,15 +1,21 @@
 import { BigNumber, ethers } from 'ethers';
-import { go, logger, PendingLog } from '@api3/airnode-utilities';
-import { BASE_FEE_MULTIPLIER, DEFAULT_RETRY_TIMEOUT_MS, PRIORITY_FEE } from '../constants';
-import { GasTarget, LogsData } from '../types';
-import { ChainOptions, PriorityFee } from '../config/types';
+import { ChainOptions, PriorityFee, GasTarget } from './types';
+import { go, logger, PendingLog } from '../';
```

Comment:
I think we should try to avoid importing from index.ts as it creates a circular dependency - which can sometimes lead to weird bugs. Better to import from the module(s) directly

Reply:
Thanks for pointing this out :+1: